### PR TITLE
lint chart with kubeval

### PR DESCRIPTION
**Pull request by nhomble**
I was motivated by the comment from @ls-jad-elkik in https://github.com/localstack/helm-charts/pull/16. I was also burnt by that bug, and I am curious what sort of testing could be done on charts. 

There does exist a github action for kubeval, but it didn't seem tailored for helm charts. I found some others online, but I wasn't too confident yet in the support since they seemed a little green. So I handled the templating in a separate step.

Curious on feedback since I am a little new to this space :)

For testing, you can see two runs:
- [fails when I just add the test](https://github.com/nhomble/helm-charts/runs/2214310193?check_suite_focus=true)
- [success when I make the containerPort fix](https://github.com/nhomble/helm-charts/actions/runs/696059084)

### CODE REVIEW

File: .github/workflows/chart-lint.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - name: template
+        run: |
+          CHARTS=charts/localstack
+          TMP=./tmp/templates
+
+          mkdir -p $TMP
+
+          cd $CHARTS
+          helm dependency update
+          cd -
```
Comment (ls-jad-elkik) [inline .github/workflows/chart-lint.yaml:26]: You can use pushd/popd here (doesn't really make a difference in this case.

```suggestion
          pushd $CHARTS
          helm dependency update
          popd
```

File: .github/workflows/chart-lint.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+          helm template $CHARTS > ${TMP}/chart-lint-template-out.yml
+          cat ${TMP}/chart-lint-template-out.yml
```
Comment (ls-jad-elkik) [inline .github/workflows/chart-lint.yaml:29]: These two lines can be shortened if you use `tee`:

```suggestion
          helm template $CHARTS | tee ${TMP}/chart-lint-template-out.yml
```

**Review by ls-jad-elkik — APPROVED** [review]
Nice!

Just added a few suggestions that don't do much at all.

File: .github/workflows/chart-lint.yaml (hunk 1 above)
Comment (nhomble) [reply .github/workflows/chart-lint.yaml:26]: I liked the `tee` suggestion :) this one seems more like personal preference.

Comment (ls-jad-elkik) [reply .github/workflows/chart-lint.yaml:26]: Indeed :) Sounds good to me!

Comment (whummer) [reply .github/workflows/chart-lint.yaml:26]: Thanks @nhomble @ls-jad-elkik . nitpick: Another possible option would be to use a subshell:
```
(cd $CHARTS; helm dependency update)
```
But I think what we have is good as well. 👍

**Review by whummer — COMMENTED** [review]
Looks great, thanks @nhomble ! 👍

**Pull request merged**